site/content/integrate/apps/quickstart/apps-debug/_index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Using /apps debug command"
```

Comment:
What about "Debugging Apps" for the title

Reply:
Building on this great idea, let's go with "Debug apps" given that we avoid gerunds across the docs set.

Reply:
In #1181 I've moved this page to the Functionality section.